theseus/src/launcher/mod.rs

```diff
@@ -440,6 +453,36 @@ pub async fn launch_minecraft(
     }
     command.envs(env_args);
 
+    // Overwrites the minecraft options.txt file with the settings from the profile
+    // Uses 'a:b' syntax which is not quite yaml
+    let options_path = instance_path.join("options.txt");
+    let mut options_string = String::new();
+    if options_path.exists() {
+        let options = io::read_to_string(&options_path).await?;
+        let options = options.split('\n');
+        for option in options {
+            let option = option.split(':').collect::<Vec<_>>();
+            if option.len() == 2 {
+                let key = option[0].trim();
+                let value = option[1].trim();
+                if let Some(value) = mc_set_options
+                    .iter()
+                    .find(|(k, _)| k == key)
+                    .map(|(_, v)| v)
+                {
+                    options_string.push_str(&format!("{}:{}\n", key, value));
+                } else {
+                    options_string.push_str(&format!("{}:{}\n", key, value));
+                }
+            }
+        }
+    } else {
+        for (key, value) in mc_set_options {
+            options_string.push_str(&format!("{}:{}\n", key, value));
+        }
+    }
+    io::write(&options_path, options_string).await?;
+
```

Comment:
I don't we should overwrite the file. I think the slider should read/write from options.txt